Carmu/Sources/Views/MyPage/PrivacyView.swift

```diff
@@ -80,54 +63,40 @@ final class PrivacyView: UIView {
     }
 
     override func draw(_ rect: CGRect) {
-        setupViews()
-        setAutoLayout()
+        setupUI()
     }
 
-    func setupViews() {
-        addSubview(privacyIntroStackView)
+    func setupUI() {
         addSubview(privacyContentView)
-
-        privacyIntroStackView.addArrangedSubview(appLogoView)
-        privacyIntroStackView.addArrangedSubview(greetingStack)
-
-        greetingStack.addArrangedSubview(greetingLabel)
-        greetingStack.addArrangedSubview(privacyDescription)
-
-        privacyContentView.addArrangedSubview(privacyContent)
-    }
-
-    // MARK: - 오토 레이아웃 설정 메서드
-    func setAutoLayout() {
-        privacyIntroStackView.snp.makeConstraints { make in
-            make.top.equalToSuperview().inset(80)
+        privacyContentView.snp.makeConstraints { make in
+            make.top.equalTo(safeAreaLayoutGuide).offset(277)
```

Comment:
safeAreaLayoutGuide도 superview 개념으로 알고 있는데 offset보다는 inset을 쓰는게 적절하지 않을까요? 🤔

```suggestion
        privacyContentView.snp.makeConstraints { make in
            make.top.equalTo(safeAreaLayoutGuide).inset(277)
```

Reply:
앗 `inset`은 `equalToSuperview`에만 쓸 수 있는건줄 알았습니다 ㅋㅋ